src/DssSpellCollateral.sol

```diff
@@ -30,99 +30,87 @@ contract DssSpellCollateralAction {
     // A table of rates can be found at
     // https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
-
+    uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978;
+    
     // --- Math ---
-    // uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant THOUSAND   = 10 ** 3;
     uint256 constant MILLION    = 10 ** 6;
     // uint256 constant BILLION    = 10 ** 9;
 
-    bytes32 constant RWA007_A                 = "RWA007-A";
-    uint256 constant RWA007_A_AUTOLINE_AMOUNT = 250 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_GAP    =  50 * MILLION;
-    uint256 constant RWA007_A_AUTOLINE_TTL    = 1 weeks;
-
     // --- DEPLOYED COLLATERAL ADDRESSES ---
-    // address constant XXX                  = 0x0000000000000000000000000000000000000000;
-    // address constant PIP_XXX              = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_JOIN_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_XXX_A       = 0x0000000000000000000000000000000000000000;
-    // address constant MCD_CLIP_CALC_XXX_A  = 0x0000000000000000000000000000000000000000;
-
-    // --- Offboarding: Current Liquidation Ratio ---
-    // uint256 constant CURRENT_XXX_A_MAT              =  XYZ * RAY / 100;
-
-    // --- Offboarding: Target Liquidation Ratio ---
-    // uint256 constant TARGET_XXX_A_MAT               =  XYZ * RAY / 100;
-
+    address internal constant RETH                 = 0xae78736Cd615f374D3085123A210448E74Fc6393;
+    address internal constant PIP_RETH             = 0xeE7F0b350aA119b3d05DC733a4621a81972f7D47;
```

Comment:
This looks like a change that wouldn't let it compile - it definitely needs fixing though.

Reply:
The deployed OSM is the old way of doing it that works with 0.5.X. Most of [dss 1.0.0](https://github.com/makerdao/dss/blob/1.0.0/src/lib.sol#L25-L42) uses this code. The `msize()` needs to be turned into a function for `solc 0.6.x` as it created a breaking change in the assembly.

It should be fine this way because it looks like the OSM was deployed with 0.5.12, so I guess the question is whether that is the intention, or do we want to use 0.6.12. I think oracles had a factory that they used to deploy standard OSM's in the past, but this seems to have been a manual deploy.

Reply:
More recent OSM's, here `PIP_RENBTC` have been deployed with the code on `master` and solc 0.6.x.

https://etherscan.io/address/0x8df8f06dc2de0434db40dcbb32a82a104218754c#code